### PR TITLE
docs: fix advice about preferred version in high availability

### PR DESCRIPTION
**Pull request by minchao**
Signed-off-by: Minchao <minchao.220@gmail.com>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: docs/operator-manual/high_availability.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ number of allowed concurrent kubectl fork/execs.
 * The controller uses Kubernetes watch APIs to maintain lightweight Kubernetes cluster cache. This allows to avoid querying Kubernetes during app reconciliation and significantly improve
 performance. For performance reasons controller monitors and caches only preferred the version of a resource. During reconciliation, the controller might have to convert cached resource from
 preferred version into a version of the resource stored in Git. If `kubectl convert` fails because conversion is not supported then controller falls back to Kubernetes API query which slows down
-reconciliation. In this case advice user-preferred resource version in Git.
+reconciliation. In this case advice to use preferred resource version in Git.
```
Comment (crenshaw-dev) [inline docs/operator-manual/high_availability.md:64]:
```suggestion
reconciliation. In this case, we advise you to use the preferred resource version in Git.
```

Would this still make sense? The grammar is a bit clearer to me.

**Review by crenshaw-dev — CHANGES_REQUESTED** [review]

File: docs/operator-manual/high_availability.md (hunk 1 above)
Comment (minchao) [reply docs/operator-manual/high_availability.md]: Thanks @crenshaw-dev
Updated.

**Review by crenshaw-dev — APPROVED** [review]
Thanks @minchao!

**Comment by crenshaw-dev** [comment]
Cherry-picked onto release-2.2 for 2.2.15, release-2.3 for 2.3.10, release-2.4 for 2.4.15, and release-2.5 for 2.5.0-rc2.

**Pull request merged**